lightning/src/ln/channel.rs

```diff
@@ -1427,6 +1437,10 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 		Ok((remote_initial_commitment_tx, localtx, remote_signature, local_keys))
 	}
 
+	fn their_funding_pubkey(&self) -> &PublicKey {
+		&self.their_pubkeys.as_ref().expect("their_funding_pubkey() only allowed after accept_channel").funding_pubkey
```

Comment:
Hmm, I generally like to force the unwrap() as close to the callsite as possible so as to avoid hiding a precondition.

Reply:
(sorry for multiple notifications, github issue)

How about if we rename this to `their_unwrapped_funding_pubkey`?

This does reduce verbosity in four callsites.

Reply:
(sorry for multiple notifications, github issue)

How about if we rename this to `their_unwrapped_funding_pubkey`?

This does reduce verbosity in four callsites.

Reply:
(sorry for multiple notifications, github issue)

How about if we rename this to `their_unwrapped_funding_pubkey`?

This does reduce verbosity in four callsites.